src/components/views/rooms/RoomList.tsx

```diff
@@ -71,13 +73,15 @@ interface IProps {
     resizeNotifier: ResizeNotifier;
     isMinimized: boolean;
     activeSpace: SpaceKey;
+    pageType: PageType;
 }
 
 interface IState {
     sublists: ITagMap;
     currentRoomId?: string;
     suggestedRooms: ISuggestedRoom[];
     feature_favourite_messages: boolean;
+    timelineRenderingType: TimelineRenderingType;
```

Comment:
What purpose does this have?

Reply:
I am not sure i added that willingly, it def be a mistake. let me remove it